packages/core/src/utils/renderThumbs.ts

```diff
@@ -0,0 +1,45 @@
+export function renderThumbs(
```

Comment:
(block)
- we need to use `kebab-case` to separate file name
- this is a specific thing about the carousel, why is it in global utils?

Reply:
done at [5c47efd](https://github.com/juntossomosmais/atomium/pull/290/commits/5c47efd5dd7f7ec230884f1a402c8fce6453d145)

Removed from global too